src/faebryk/libs/app/erc.py

```diff
@@ -87,7 +88,8 @@ def simple_erc(G: Graph):
     ]
 
     if unresolved_voltage:
-        raise ERCFaultElectricPowerUndefinedVoltage(unresolved_voltage)
+        with downgrade(ERCFaultElectricPowerUndefinedVoltage):
```

Comment:
This doesn't make sense until we have the params working well.

By default, we should not fail builds because they're uncheckable - only if we know they're wrong